databroker/src/grpc/kuksa_val_v2/val.rs

```diff
@@ -1907,27 +2116,37 @@ mod tests {
                         }),
                         None => None,
                     },
-                }),
-            });
-
-            request
-                .extensions_mut()
-                .insert(permissions::ALLOW_ALL.clone());
-            match broker.publish_value(request).await {
-                Ok(response) => {
-                    // Handle the successful response
-                    let publish_response = response.into_inner();
+                },
+            );
 
-                    // Check if there is an error in the response
-                    assert_eq!(publish_response, proto::PublishValueResponse {});
+            let f = false;
```

Comment:
What's with the `let f = false;`?

Reply:
We have been dragging this kind of assertion for a while I think.